### PR TITLE
fix(toolbar): make suggestion diffs theme-friendly

### PR DESCRIPTION
**Pull request by queengooborg**
## Summary

This PR updates the backgrounds of the `<ins>` and `<del>` elements inside of the flaws panel to make them theme-friendly.  This resolves an accessibility (contrast) issue.

---

### CODE REVIEW

**Comment by queengooborg** [comment]
CC @caugner @fiji-flo -- this change is a very simple one that fixes a major accessibility issue.

**Comment by caugner** [comment]
@queengooborg Can you please fill out the pull request template, including "How did you test this change?"? 🙏

**Comment by queengooborg** [comment]
I updated the issue description to match the template, but I've pretty much already filled out all of the applicable fields from the template.  The "How did you test this change?" doesn't apply because it's a UI update only.  .-.

File: client/src/ui/base/_themes.scss [hunk 1]
```diff
@@ -227,14 +229,8 @@
...
-  --background-critical: #{color.adjust(
-      $mdn-theme-light-icon-critical,
-      $alpha: -0.9
-    )};
-  --background-success: #{color.adjust(
-      $mdn-theme-light-icon-success,
-      $alpha: -0.9
-    )};
+  --background-del: #{color.adjust($mdn-theme-light-icon-critical, $alpha: -0.5)};
+  --background-ins: #{color.adjust($mdn-theme-light-icon-success, $alpha: -0.5)};
```
Comment (caugner) [inline client/src/ui/base/_themes.scss:233]: We're still using `--background-critical` and `--background-success`.

**Review by caugner — CHANGES_REQUESTED** [review]

**Review by caugner — APPROVED** [review]
We (@argl @fiji-flo @caugner) took a look and approve the changes.

**Pull request merged**